src/odemis/acq/feature.py

```
@@ -819,12 +749,15 @@ def _export_data(self, feature: CryoFeature, data: List[model.DataArray]) -> Non
         filename = add_feature_info_to_filename(feature, self.filename)
 
         # add feature name to the data description
-        for d in data:
+        for i, d in enumerate(data):
```

Ruff (0.15.10)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)